Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -928,6 +928,7 @@ public decimal CalculateOrderQuantity(Symbol symbol, double target)
         /// <returns>Order quantity to achieve this percentage</returns>
         public decimal CalculateOrderQuantity(Symbol symbol, decimal target)
         {
+            if (!Securities.ContainsKey(symbol)) return 0; // check if symbol exists in Securities
```

Comment:
If it returns zero when the symbol is not in Securities, we hide this user error:
```This asset symbol ( ) was not found in your security list. Please add this security or check it exists before using it with 'Securities.ContainsKey(" ")'```

Reply:
I have removed this check,
commit c4f54610232a296fc5f0985f7b23a75b443492a6